src/main/java/seedu/address/logic/commands/GetPrev3MonthFeeCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.fee.Month;
+import seedu.address.model.fee.Year;
+
+/**
+ * Gets current month and previous 2 months expected monthly fee based on the current list of sessions
+ */
+public class GetPrev3MonthFeeCommand extends Command {
+    public static final String COMMAND_WORD = "3Month_Fee";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows the total expected monthly fee"
+        + "for this month + previous 2 months";
+
+    /**
+     * Gets the local date time format of the month and year combined
+     * @return LocalDateTime of the month and year combined
+     */
+    private LocalDateTime getLocalDate(Month month, Year year) {
+        requireAllNonNull(month, year);
+        return LocalDateTime.of(year.getYear(), month.getMonth(), 1, 0, 0);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        LocalDateTime currMonthYear;
+        LocalDateTime nextMonthYear;
+        StringBuilder resultString = new StringBuilder("Total Monthly fee for past 3 months is:");
```

Comment:
Could possibly remove "total", just let it be "Monthly fee for the past 3 months"